devel/go-bindata/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/shuLhan/go-bindata 3.4.0 v
+
+description         A small utility which generates Go code from any file. \
+                    Useful for embedding binary data in a Go program.
+
+long_description    {*}${description} The file data is optionally gzip \
+                    compressed before being converted to a raw byte slice.
+
+categories          devel
+license             public-domain # Creative Commons Zero v1.0 Universal
```

Comment:
Tcl does not have middle-of-line comment capability, so that entire value is interpreted as the license:
```
$ port info | grep License
License:              public-domain and # and Creative and Commons and Zero and v1.0 and Universal
```
Comments must always begin at the beginning of the line, excepting whitespace.
```suggestion
# Creative Commons Zero v1.0 Universal
license             public-domain
```

Reply:
My fault - fixed